src/psyclone/tests/psyir/transformations/psy_data_test.py

```diff
@@ -271,3 +271,64 @@ def test_psy_data_invokes_gocean1p0():
     # variables and region names are created:
     code_again = str(invoke.gen()).replace("\n", "")
     assert code == code_again
+
+
+# -----------------------------------------------------------------------------
+def test_psy_data_options():
```

Comment:
This is not a test of transformations. Maybe it should be under tests/psyir/nodes instead?

Reply:
OK, considering #693 I've split this into two different files, one for the transforms, one for the node. Now it is sometimes different to decided which goes where: E.g. we use a transform to insert the node, then check the output of `gen()` - it is a bit of both, but I think I did a reasonable division there.

Note that none of the two files covers 100% of the lines (though I added/duplicated a few tests to cover at least some basic functionality - aka low hanging fruits ;) ). See some additional comments in #693.

Reply:
Thanks for the valuable feedback in #693. I can see your point - it is going to be hard to do this separation for the nodes introduced by transformations :-(